Automattic/jetpack
# [Package] Backup: Request review after successful interactions

#### Changes proposed in this Pull Request:

Add a message requesting a plugin review after a successful restore.
The message will be displayed using the Contextual Upgrade Trigger (CUT) component.

#### Before:
<img width="1440" alt="WP Admin - Regular Use0" src="https://user-images.githubusercontent.com/16329583/176874131-d4788173-efb2-420d-b6c5-ce48fe006554.png">


#### After:
![WP Admin - Regular Use](https://user-images.githubusercontent.com/16329583/176874119-77bb3266-794d-48ed-92e9-d61352d2691a.png)


#### To do:
- [x] Add component to Backup admin page
- [x] Create redirect to review page
- [x] Check how to get recent restores ( may require endpoint )
- [x] Check how to get Backup subscription time ( will be used for other conditions to display message )
- [x] Write logic to verify eligibility to display message
- [ ] Add styling for component
- [x] Add tracking (if necessary)

#### Other information:
#### Jetpack product discussion

P2 discussion about this change here:
https://wp.me/pbuNQi-3bh

#### Does this pull request change what data or activity we track or use?
Not for the moment, but we may track the clicks on the reviews button.


#### Testing instructions:





- Create Jurassic Ninja site. You can use this link:
https://jurassic.ninja/create?jetpack-beta&branches.jetpack-backup=add/review-request&wp-debug-log
You can add &nojetpack at the end to run it without the JP plugin.
- Buy a Backup subscription and complete the initial backup
- Go to cloud.jetpack.com, add the SSH credentials and run a restore
- Once the restore is completed, verify that the message asking for a review is visible and that the link points to the plugin review page
